tests/template.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: assisted-service-mcp-eval-test
+objects:
+- apiVersion: batch/v1
+  kind: Job
+  metadata:
+    name: assisted-service-mcp-eval-test-${JOB_ID}
+    labels:
+      app: assisted-service-mcp-eval-test
+  spec:
+    template:
+      metadata:
+        labels:
+          app: assisted-service-mcp-eval-test
+      spec:
+        restartPolicy: Never
+        containers:
+        - name: assisted-service-mcp-eval-test
+          image: ${IMAGE_NAME}
```

Comment:
Which image will it be?

Reply:
the same image we use in the assisted-chat tests:
quay.io/redhat-user-workloads/assisted-installer-tenant/assisted-chat-test-image-saas-main/assisted-chat-test-image-saas-main